scripts/benchmark_ipa.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Generate stage-wise report after running IPA bench
+#
+# To use, supply the number of rows you want to run benchmarking on
+
+num_rows=$1
+if [ "$#" -ne 1 ]; then
+    echo "Please supply query size as an argument to the script"
+    exit
+fi
+
+temp_file=".benchmark_result$num_rows.csv"
+RUST_LOG=ipa=DEBUG cargo bench --bench oneshot_ipa --features="enable-benches" --no-default-features -- -n $num_rows > $temp_file
+ 
+until_now_records=0
+until_now_bytes=0
+
+echo "Step,Bytes sent,Records Sent"
+step_mapping=( 
+        "mod_conv_breakdown_key\|mod_conv_match_key:Verify"
+        "apply_sort_permutation\|gen_sort_permutation_from_match_keys:Sort"
+	"accumulate_credit\|compute_helper_bits:Attribution"
+  	"user_capping:Capping"
+        "check_times_credit\|compute_equality_checks:Aggregation"
+)
+        
+
+for step in "${step_mapping[@]}" ; do
+    key=${step%%:*}
```

Comment:
```suggestion
    key="${step%%:*}"
```
Variable expansions should be quoted.